lib/defaults.js

```diff
@@ -0,0 +1,60 @@
+var defaults = {
+  jquery: {
+    style: 'jquery',
+    noSemver: false,
```

Comment:
I added for the following scenario: 

1. I do the Angular PR and tell the developers there that, for example, commitplease allows `[Tmp]: this is a temp commit`.
2. They dislike it and tell me they do not want it at all.
3. I have two options: fix the code and do a quick npm bump or fix the PR changing the setting.
4. So, I decided to make many settings. Probably bad call indeed.

Do I remove those then?